config/migrations/question-is-subsidy-collaboration-concept-scheme.sparql

```diff
@@ -0,0 +1,20 @@
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/application> {
+<http://lblod.data.gift/concept-schemes/af23feb1-a876-4834-9427-947f93b776f2> <http://mu.semte.ch/vocabularies/core/uuid> "af23feb1-a876-4834-9427-947f93b776f2" ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#ConceptScheme> ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Possible answers to the question: Is this subsidy-application an application for a collaboration"@en .
+
+<http://lblod.data.gift/concepts/70fea9a0-9ccc-48e6-bc3e-269a7f070264> <http://mu.semte.ch/vocabularies/core/uuid> "70fea9a0-9ccc-48e6-bc3e-269a7f070264" ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class>, <http://www.w3.org/2004/02/skos/core#Concept> ;
+	<http://www.w3.org/2004/02/skos/core#definition> "The subsidy-application is not for a collaboration"@en ;
+	<http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/af23feb1-a876-4834-9427-947f93b776f2> ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Nee"@nl .
+
+<http://lblod.data.gift/concepts/9ffe7dd4-42b7-4af3-bdba-ccd080195715> <http://mu.semte.ch/vocabularies/core/uuid> "9ffe7dd4-42b7-4af3-bdba-ccd080195715" ;
+	<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class>, <http://www.w3.org/2004/02/skos/core#Concept> ;
+	<http://www.w3.org/2004/02/skos/core#definition> "The subsidy-application is for a collaboration"@en ;
+	<http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/af23feb1-a876-4834-9427-947f93b776f2> ;
+	<http://www.w3.org/2004/02/skos/core#prefLabel> "Ja"@nl .
+
```

Comment:
Shouldn't be there a more generic `concept-scheme` for this? I know we can't change this as the `concept-scheme` is used like this but maybe something to think about.

Reply:
Also not something we should do in this PR